### DISCUSSION TITLE
How do I access my Liked Songs playlist?

### DISCUSSION THREAD

**Discussion by beedell-roke** [kraxarn/spotify-qt#234] [Q&A] [answered]
It's not visible:

![Screenshot_20250216_215136](https://github.com/user-attachments/assets/dfd98239-48aa-40bc-81df-00ae672a9337)

**Comment by beedell-roke** [comment]
> https://github.com/kraxarn/spotify-qt/discussions/234#discussion-5856150

I've filed this as an issue at https://github.com/kraxarn/spotify-qt/issues/235#issue-1995777023.

**Comment by kraxarn** [comment, marked as answer]
It's in "Liked tracks".

**Reply by beedell-roke** [reply]
> https://github.com/kraxarn/spotify-qt/discussions/234#discussioncomment-7673822

Hahaha, thanks, @kraxarn. I didn't notice.
